app/views/shared/_footer_lite.html.slim

```diff
@@ -1,12 +1,15 @@
-footer.footer.bg-navy.white
-  .container.py2.h6
+footer.footer.bg-light-blue.sm-bg-navy
+  .container.h6.py1.px2.lg-px0
     .clearfix
-      .sm-col-right.caps.pt1.pb2.sm-pt0.sm-pb0.center.caps
+      .col-right.caps.py0.center.caps
```

Comment:
We can remove the `py0.center.caps` from this since it's no longer stacked on mobile and `caps` is already listed once.